config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
`:passw`? Typo or is this matching on start of word or sthn?

Reply:
Also might want to check `token` since we might actually want that from `PostRedirect`?

Reply:
Ah yeah I was going to go back to check that, that is the new default from Rails. I guess they are matching start of params keys. Will check

Reply:
Before:
```
Processing by Users::SessionsController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"", "user_signin"=>{"email"=>"bob@localhost", "password"=>"[FILTERED]"}, "token"=>"jof82ifzn86eshb82c5", "modal"=>"", "commit"=>"Sign in", "locale"=>"en"}
```

After:
```
Processing by Users::SessionsController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"[FILTERED]", "user_signin"=>{"email"=>"bob@localhost", "password"=>"[FILTERED]"}, "token"=>"[FILTERED]", "modal"=>"", "commit"=>"Sign in", "locale"=>"en"}
```

Reply:
> Also might want to check :token

Yeah we use token other places too like project invites so I think we should drop `:token`.

Reply:
Updated in d16de64